src/js/views/draftScouting.js

```diff
@@ -26,7 +26,10 @@ async function addSeason(season, tid) {
         const pa = playersAll[i];
 
         // Abbreviate first name to prevent overflows
-        pa.name = `${pa.firstName.split(" ").map(s => s[0]).join(".")}. ${pa.lastName}`;
+        const names = pa.firstName.split(/[ .-]/);
+        const abbrev = names.map(s => {return s === "" ? "" : `${s[0]}.`;});
+        const concat = abbrev.join("");
+        pa.name = `${concat} ${pa.lastName}`;
```

Comment:
As of right now, this section is the only place that ever uses abbreviated names. In the event that it becomes more widespread, how would you feel about a "player abbreviation" entry being added to player.filter?